tray/src/com/dvt/main/creational/singleton/Database.java

```diff
@@ -0,0 +1,18 @@
+<<<<<<< HEAD
+package com.dvt.creational.singleton;
+=======
+package creational.singleton;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.singleton;
=======
package creational.singleton;
>>>>>>> main
```